gammapy/stats/variability.py

```diff
@@ -303,6 +304,8 @@ def TimmerKonig_lightcurve_simulator(
         Number of points in the output time series.
     spacing : `~astropy.units.Quantity`
         Sample spacing, inverse of the sampling rate. The units are inherited by the resulting time axis.
+    leakage_protection : int
```

Comment:
Add the used default value in the docstring and precise that it should be larger than 1
I think also that using a float is more efficient for this purpose, there is no need to create artificially a too large number of bins.... If accepted, then a default value of 1.2 or 1.5 might be wise (maybe some tests are needed to derive a 'reasonable' default value)